src/routes/admin.ts

```diff
@@ -0,0 +1,66 @@
+import express from 'express';
+import Admin from '@/services/Admin';
+import Utils from '@/utils';
+
+const router: express.Router = express.Router();
+const admin = new Admin();
+
+// TODO get last x entries, or entries from x to y with total size of 100
+// TODO get open jobs from query
+// TODO get statistics
+
+router.get('/faildJobsByReceiver/:receiver', async (req, res) => {
+	// TODO add shd key verification
+	// validate regex
+	if (Utils.parametersMissing(['receiver'], req.params, res)) {
+		return;
+	}
+
+	try {
+		const failedJobs = await admin.getFailedJobByReceiver(req.params.receiver);
+		res.json({
+			failedJobs,
+		});
+	} catch (err) {
+		res.status(400).send(err.message);
+	}
+});
+
+router.get('/failedJobs/', async (req, res) => {
+	try {
+		// from and until schema = 2020-09-09T08:50:51.181Z
```

Comment:
Use date as type for from and until, I would parse them from the query into a new object having parameters typed and verified (from<until) and both given.